docs/development.rst

```diff
@@ -26,19 +52,52 @@ Quickstart
 
   $ git submodule update --init --recursive
 
+.. - Install the  development :ref:`basic requirements <sec_installation_system_requirements>`.
+.. - Install the Python development requirements using
+..   ``pip install -r python/requirements/development.txt``.
+
+- To work on the Python code, ``cd`` into the ``python`` directory
+- Build the low level module by running ``make`` in the project root.
+- Run the tests to ensure everything has worked: ``python3 -m nose -vs``. These should
+  all pass.
+- Make your changes in a local branch, and open a pull request on GitHub when you
+  are ready. Please make sure that (a) the tests pass before you open the PR; and
+  (b) your code passes PEP8 checks (see below for a git commit hook to ensure this
+  happens automatically) before opening the PR.
+- See the `tskit documentation <https://tskit.readthedocs.io/en/latest/development.html#github-workflow>`_
+  for more details on the recommended GitHub workflow.
+
+
+-----------------
+Python Quickstart
+-----------------
+
+- Make a fork of the tskit repo on `GitHub <http://github.com/tskit-dev/tskit>`_
+- Clone your fork into a local directory, making sure that the **submodules
+  are correctly initialised**::
+
+  $ git clone git@github.com:your-github-username/tskit.git --recurse-submodules
+
+  For an already checked out repo, the submodules can be initialised using::
+
+  $ git submodule update --init --recursive
+
   (note that on Windows you may have to run this as administrator, in order to create the
   correct symlinks)
-- Install the Python development requirements using
-  ``pip install -r python/requirements/development.txt``.
-- Build the low level module by running ``make -C python``.
-- Run the tests to ensure everything has worked: ``python -m nose -vs python``. These
+
+- Install the pre-commit checks: ``pre-commit install`` (these are used
```

Comment:
need to say to install the python package prereqs before this (and maybe preprend "pre-commit is a python package, not a git command" to the parenthetical note - it confused me at first - is this a typo for "git pre-commit"?)